ripple/src/main/webapp/script.js

```diff
@@ -129,19 +129,14 @@ function searchInput(keyPress) {
     var searchString = document.getElementById("search-bar").value;
     console.log(searchString);
     localStorage.setItem("searchString", searchString);
-    localStorage.setItem("searchStringRaw", )
+    localStorage.setItem("searchStringRaw", );
```

Comment:
Ignore syntax error. Smruthi to update in next merge.

Reply:
Should be resolved in the pull request I just merged!